lenskit-core/src/main/java/org/grouplens/lenskit/data/sql/JDBCRatingDAO.java

```diff
@@ -136,8 +136,14 @@ public JDBCRatingDAO(Connection dbc, SQLStatementFactory sfac, boolean close) {
         userStatement = new CachedPreparedStatement(dbc, statementFactory.prepareUsers());
         itemStatement = new CachedPreparedStatement(dbc, statementFactory.prepareItems());
         for (SortOrder order : SortOrder.values()) {
-            eventStatements.put(order, new CachedPreparedStatement(dbc,
-                                                                   statementFactory.prepareEvents(order)));
+            switch (order) {
+            case ASCENDING:
```

Comment:
I'm not going to claim I understand whats going on here, but this doesn't make sense in context, especially a break instead of a continue.